From 517fa766782421302da827278ca17e6b2ad57da3 Mon Sep 17 00:00:00 2001
From: Sumit Bose <sbose@redhat.com>
Date: Fri, 21 Feb 2020 14:06:16 +0100
Subject: [PATCH 7/7] service: use "kerberos method" "secrets and keytab"

When using Samba with Winbind the host password stored in secrets.tdb is
still important so the "secrets and keytab" should be the preferred
"kerberos method".

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1801195
---
 service/realm-samba.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/service/realm-samba.c b/service/realm-samba.c
index e2a3608..4940b38 100644
--- a/service/realm-samba.c
+++ b/service/realm-samba.c
@@ -200,7 +200,7 @@ on_join_do_winbind (GObject *source,
 		                         "template shell", realm_settings_string ("users", "default-shell"),
 		                         "netbios name", computer_name,
 		                         "password server", enroll->disco->explicit_server,
-		                         "kerberos method", "system keytab",
+		                         "kerberos method", "secrets and keytab",
 		                         NULL);
 	}
 
-- 
2.25.1

